### ISSUE TITLE
Gallery issues

### ISSUE THREAD

**Issue by TheLoombot** [TheLoombot/99-reddits#84]
Two behaviors since fixing #82 ... (observed on my iPhone X hardware).

First one:

1. Open a gallery view with enough thumbnails to require scrolling 
2. Scroll down to bottom, observe that "MOAR" button is cut off 
3. You can still tap it, but it's cut off in the "resting position:"
4. Note that much like in #82 ... if you tap into a photo and come back, the problem goes away

![2017-11-18 10 08 22](https://user-images.githubusercontent.com/4617505/32981895-f23712f2-cc48-11e7-85f5-f608f9fc2cb8.png)

Second one:

We've seen this before, but I think the same layout issue results in it... basically, the tab bar at the bottom of the screen isn't flush and the thumbnails "peek" through when you scroll:

![2017-11-18 10 08 10](https://user-images.githubusercontent.com/4617505/32981900-1f7f107a-cc49-11e7-8bbb-7d8ebf4d6a35.png)

**Comment by pietrorea** [comment]
https://github.com/TheLoombot/99-reddits/pull/87

So the 1pt "peek" is a workaround that I added to un-jank the bottom toolbar on iPhone X. I removed the 1pt space, and it now looks like this, which I think is acceptable (the space is too big).

![simulator screen shot - iphone x - 2017-11-18 at 17 19 00](https://user-images.githubusercontent.com/1264371/32985267-1253c8dc-cc85-11e7-8bd9-d927b1b89200.png)

UIToolbar is having all sorts of problems on the iPhone X. I wrote up a radar for this. I hope we hear back: http://www.openradar.me/radar?id=5024542300831744

The other issue should also be solved
